### ISSUE TITLE
duplicated feedstock

### ISSUE THREAD

**Issue by 12rambau** [conda-forge/jupyter_sphinx-feedstock#27]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

@conda-forge/python I am a new maintainer of the jupyter-sphinx package and by setting up the badges in the readme file I realized there are 2 feedstocks wired to the same package. 

It was mentioned a long time ago in https://github.com/jupyter/jupyter-sphinx/issues/148 but it seems nobody took actions. 

we have: 
- https://github.com/conda-forge/jupyter_sphinx-feedstock
- https://github.com/conda-forge/jupyter-sphinx-feedstock

I would like to only keep the "jupyter-sphinx" one to use the same naming convention as in the pypi package. what process should I follow to make this one disapear ? 

### Installed packages

```shell
NA
```

### Environment info

```shell
NA
```

**Comment by akhmerov** [comment]
A while ago I asked what is the way to go on gitter, and the advice was to provide `jupyter_sphinx` as an alias package for `jupyter-sphinx` and maintain both via `jupyter-sphinx` feedstock. Once this is done, this feedstock would be removed. Note that both packages have comparable amounts of downloads on conda-forge, so deleting one would likely be disruptive.

**Comment by ocefpaf** [comment]
> A while ago I asked what is the way to go on gitter, and the advice was to provide `jupyter_sphinx` as an alias package for `jupyter-sphinx` and maintain both via `jupyter-sphinx` feedstock. Once this is done, this feedstock would be removed. Note that both packages have comparable amounts of downloads on conda-forge, so deleting one would likely be disruptive.

Correct. That is the best path forward for dealing with `-` vs `_` in package names.
